package.json

```
@@ -4,11 +4,13 @@
         "build-master": "rm -rf ./master/build && tsc -p ./master",
         "build-local": "rm -rf ./local/build && tsc -p ./local",
         "lint": "eslint . --ext .ts",
+        "fix": "eslint . --ext .ts --fix",
```

```suggestion
        "fix": "eslint . --ext .ts --fix --cache",
```

You can tell eslint to cache unchanged files, which improves speed significantly. You will have to gitignore `.eslintcache` if you decide to implement this.

See [the docs for this option](https://eslint.org/docs/latest/use/command-line-interface#options)